README.md

```diff
@@ -67,9 +67,6 @@ Configuration lives in `~/.nixpkgs/darwin-configuration.nix`. Check out
     [ pkgs.vim
     ];
 
```

Comment:
Nit: Stray blank line left over. Also, maybe we should leave `nix.package = pkgs.nix;` in the examples so that people know how to override it?

Reply:
I've readded the example but changed it to `nix.package = pkgs.nixVersions.nix_2_18;` to make it more clear what the line is, because I think before it wasn't clear that `pkgs.nix` was already the default and the line was unnecessary